### PR TITLE
add languageId default value

### PR DESCRIPTION
**Pull request by chiiia12**
## Issue
- None

## Overview (Required)
- AfterParty's language text is empty, I add default value to 'languageId' in SessionViewModel.

## Links
-

## Screenshot
Before | After
:--: | :--:
device-2017-02-06-205946 |device-2017-02-06-205835

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java [hunk 1]
```diff
@@ -67,9 +67,8 @@
         if (session.room != null) {
             this.roomName = session.room.name;
         }
-        if (session.lang != null) {
-            this.languageId = session.lang.toUpperCase();
-        }
+       
+        this.languageId = session.lang != null ? session.lang.toUpperCase() : Session.LANG_EN_ID;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: I think it's better to hide language tag and speaker name.
Can you add `languageVisibility` in this class and bind it to `view_session.xml`? 🙇

**Comment by konifar** [comment]
Commented! Thanks for your contribution! 👏

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java (hunk 1 above)
Comment (chiiia12) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: Thanks reviewing!
I fix it. when `session.lang` is null,  make `txt_language`  hide.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: Perfect 💯 Thanks!!

**Comment by konifar** [comment]
LGTM!

**Pull request merged**